### PR TITLE
TW-10: Add default limits for Temporal Cloud

### PR DESCRIPTION
**Pull request by djmagee**
## What does this PR do?

- Adds default limits for Temporal Cloud. (Info gleaned from ancient PR #1639.)
- Removes inline-code style (backticks) from link text, per recent style-guide change.

### CODE REVIEW

**Review by flossypurse — APPROVED** [review]

File: kb/temporal-platform-limits-sheet.md [hunk 1]
```diff
@@ -43,3 +43,15 @@ Hard limits fail with an error; soft limits produce a warning log on the server
...
     - `limit.numPendingChildExecutions.error`
 - [Search Attributes maximums](/visibility/#search-attributes-maximums)
+
+**Default limits for Temporal Cloud**
+
+- **Account level**
+  - Namespaces: 10
+  - Prometheus endpoint Retention Period: 1 month
+- **Namespace level**
+  - Actions per second: 200
```
Comment (spipe007) [inline kb/temporal-platform-limits-sheet.md:53]: with possibility to spike up to 2x

Comment (spipe007) [inline kb/temporal-platform-limits-sheet.md:53]: with possibility to spike up to 2x

Comment (rohitgup14) [reply kb/temporal-platform-limits-sheet.md:53]: Can we update that actions per second is peak

File: kb/temporal-platform-limits-sheet.md [hunk 2]
```diff
@@ -43,3 +43,15 @@ Hard limits fail with an error; soft limits produce a warning log on the server
...
+  - Certificates: 32-KB payload or 16 certificates, whichever is smaller
+  - Concurrent Task pollers: 2,000 (configurable to a maximum of 100,000)
+  - Custom Search Attributes: 20; maximum of 5 for text
```
Comment (rohitgup14) [inline kb/temporal-platform-limits-sheet.md:56]: 5 for text but AFAIR it is 5 per type as well. Can we confirm before posting ?

Comment (djmagee) [reply kb/temporal-platform-limits-sheet.md:56]: That does sound familiar; I'll check.

File: kb/temporal-platform-limits-sheet.md [hunk 3]
```diff
@@ -43,3 +43,15 @@ Hard limits fail with an error; soft limits produce a warning log on the server
...
+  - Retention Period: 7 days (configurable to a maximum of 90 days)
```
Comment (rohitgup14) [inline kb/temporal-platform-limits-sheet.md:57]: IMO we should state that Retention Period: 7 days ( configurable between 1-90 days )

File: kb/temporal-platform-limits-sheet.md (hunk 1 above)
Comment (djmagee) [reply kb/temporal-platform-limits-sheet.md:53]: @rohitgup14: Which of the following would be mo'better?
- Peak actions per second: 200 (can spike to 400)
- Actions per second (peak): 200 (can spike to 400)
- Some other variation

File: kb/temporal-platform-limits-sheet.md (hunk 2 above)
Comment (djmagee) [reply kb/temporal-platform-limits-sheet.md:56]: According to Candace, it's 20 per type except for text, which is 5.

**Review by rohitgup14 — APPROVED** [review]
LGTM

**Pull request merged**